docs/debezium-aws-rds-config-scripts

```diff
@@ -0,0 +1,361 @@
+
+10. Change table so that it can ***support replication***
+
+    `testdb=> ALTER TABLE customers REPLICA IDENTITY FULL;`
+
+11. ***Insert some records*** into the source table
+
+----
+    testdb=> insert into customers values(default, 'fatih1', 'kose1', 'fatih1.kose@example.com');
+    testdb=> insert into customers values(default, 'fatih2', 'kose2', 'fatih1.kose@example.com');
+----
+
+### 1.3 Prepare Debezium Test Environment And Complete Configuration On Linux
+
+1. Create a directory under user home directory 
+
+    `[ec2-user@hostname ~] $ mkdir ~/debezium`
+
+2. Download the sample config scripts from ***https://github.com/fatihmehmetkose/debezium-aws-rds-config-scripts*** and copy them into that directory
+
+    `[ec2-user@hostname ~] $ cp *.zip debezium/`
+
+3. Enter into that directory
+
+    `[ec2-user@hostname ~] $ cd debezium`
+
+4. ***export DEBEZIUM_VERSION*** environment variable
+
+    `[ec2-user@hostname ~] $ export DEBEZIUM_VERSION=0.7.1`
+
+5. ***Build docker-compose files*** named docker-compose.yaml (default name)
+
+    `[ec2-user@hostname ~] $ docker-compose build`
+
+6. ***Start the docker*** containers. Applications running on containers are ***Zookeper, Apache Kafka, Debezium Kafka Connector, PostgreSQL db***. 
+
+    `[ec2-user@hostname ~] $ docker-compose up`
+
+7. Create a table named ***customers*** on target or sink PostgreSQL db
+
+    `[ec2-user@hostname ~] $ docker-compose -f docker-compose.yaml exec postgres bash -c 'psql -U $POSTGRES_USER $POSTGRES_DB -c "CREATE TABLE customers (id SERIAL,  first_name VARCHAR(255) NOT NULL,  last_name VARCHAR(255) NOT NULL,  email VARCHAR(255) NOT NULL,  PRIMARY KEY(id));"'`
+
+8. ***Start target PostgreSQL DB connector*** running in a container
+
+    `[ec2-user@hostname ~] $ curl -i -X POST -H "Accept:application/json" -H  "Content-Type:application/json" http://localhost:8083/connectors/ -d @jdbc-sink.json`
+
+9. ***Start source AWS RDS PostgreSQL DB connector***
+
+    `[ec2-user@hostname ~] $ curl -i -X POST -H "Accept:application/json" -H  "Content-Type:application/json" http://localhost:8083/connectors/ -d @source-postgres.json`
+
+10. ***Verify*** that the connector definitions are added to the debezium connector
+    
+	`[ec2-user@hostname ~] $ curl -H "Accept:application/json" localhost:8083/connectors/`
+
+11. ***Verify*** that the PostgreSQL database has the same content as in AWS RDS PostgreSQL DB:
+
+    `[ec2-user@hostname ~] $ docker-compose -f docker-compose.yaml exec postgres bash -c 'psql -U $POSTGRES_USER $POSTGRES_DB -c "select * from customers"'`
+
+12. Optionally, if you wanna display the current db name, run the following command 
+
+    `[ec2-user@hostname ~] $ docker-compose -f docker-compose.yaml exec postgres bash -c 'psql -U $POSTGRES_USER $POSTGRES_DB -c "SELECT current_database();"'`
+
+13. Optionally, If you wanna login into PostgreSQL DB container with an of "f86dfa1bba34", run the following command 
```

Comment:
with a *name* of "...". Would also be nice to define a name before, when starting the container.

Reply:
Ok